### DISCUSSION TITLE
Sudo a password is required in container (Installing chrome driver)

### DISCUSSION THREAD

**Discussion by hitchon1** [apache/airflow#28389] [Q&A] [answered]
Hi all, 

I have been trying to install chrome into my docker container so I can use it for webscraping with selenium & beautifulSoup

when I run docker-compose build I get the issue of "sudo: a password is required" I am not sure what the password would be for the docker terminal anyone have issues on know how to fix this so I can install chrome. 

Thank you!

**In my dockerFile I have the following:**

`
FROM apache/airflow:2.5.0
COPY requirements.txt /
RUN pip install --no-cache-dir -r /requirements.txt

RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add - \
    sudo sh -c 'echo "deb https://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
    sudo apt-get -y update \
    sudo apt-get -y install google-chrome-stable

RUN sudo apt-get -y install chromium-chromedriver
`

**Comment by Taragolis** [comment, marked as answer]
- [Building the image](https://airflow.apache.org/docs/docker-stack/build.html#building-the-image)
- [Customizing the image ](https://airflow.apache.org/docs/docker-stack/build.html#customizing-the-image)
